css/index.css

```diff
@@ -2007,6 +2007,14 @@ Logic Interface CSS
     border-color: white;
 }
 
+.ru-canvas {
+    position: absolute;
```

Comment:
Should have `pointer-events: none` for a bit of paranoia

Reply:
Great point – too many bugs have been caused in the past by forgetting to include that style